code/game/objects/items/devices/instruments.dm

```diff
@@ -164,3 +198,60 @@
 	throw_range = 15
 	hitsound = 'sound/items/bikehorn.ogg'
 
+//spooky stuff
+/obj/item/device/instrument/brass/trumpet/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/saxophone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/saxophone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/trombone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/Cn4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+//spooky procs
+/obj/item/device/instrument/brass/proc/spectral_attack(mob/living/carbon/C, mob/user)
+	if(ishuman(user)) //this weapon wasn't meant for mortals.
+		var/mob/living/carbon/human/U = user
+		if(!istype(U.dna.species, /datum/species/skeleton))
+			U.adjustStaminaLoss(25) //Extra Damage
+			attack(user)
+			U.Knockdown(30)
+			to_chat(U, "<span class= 'danger'> Your ears weren't meant for this spectral sound.</span>")
+			return ..()
+
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(istype(H.dna.species, /datum/species/skeleton))
+			return ..() //undead are unaffected by the spook-pocalypse.
+		if(istype(H.dna.species, /datum/species/zombie))
+			H.adjustStaminaLoss(25) //zombies can't resist the doot
+		C.Knockdown(15)

```

Comment:
One hit stunlock convert that isn't counter able is shit.

Reply:
I could alter it to be simply raw stamina loss.

Reply:
Done. They will no longer stun living creatures.